src/PrestaShopBundle/Form/Admin/Sell/Product/Combination/CombinationManagerType.php

```diff
@@ -45,9 +47,27 @@
  */
 class CombinationManagerType extends TranslatorAwareType
 {
+    /**
+     * @var FeatureInterface
+     */
+    private $multiStoreFeature;
+
+    /**
+     * @param FeatureInterface $multiStoreFeature
+     */
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        FeatureInterface $multiStoreFeature
+    ) {
+        parent::__construct($translator, $locales);
+        $this->multiStoreFeature = $multiStoreFeature;
+    }
+
     public function buildView(FormView $view, FormInterface $form, array $options)
     {
         $view->vars['productId'] = $options['product_id'];
+        $view->vars['isMultiStoreActive'] = $this->multiStoreFeature->isActive();
```

Comment:
FYI if you remember @jolelievre we had some discussion in private about some issue when not converting bool to int causing troubles in data attr in some other PR, but after all it seems to work fine, probably it was some other issue there. (so when i pass bool to data attr, if its false it will be rendered like `data-is-multi-store-active=""` when its true it will be `data-is-multi-store-active="1"`)

Reply:
I do remember, it works here because you manually add the data attribute (hard-coded in twig template). The case we had was when you use the form fields options and set its `attr` option, then it is rendered by the `widget_attributes` block in twig which interprets boolean values like `checked="checked"` or `disabled="disabled"`

Reply:
```twig
{% block attributes -%}
    {%- for attrname, attrvalue in attr -%}
        {{- " " -}}
        {%- if attrname in ['placeholder', 'title'] -%}
            {{- attrname }}="{{ translation_domain is same as(false) or attrvalue is null ? attrvalue : attrvalue|trans(attr_translation_parameters, translation_domain) }}"
        {%- elseif attrvalue is same as(true) -%}
            {{- attrname }}="{{ attrname }}"
        {%- elseif attrvalue is not same as(false) -%}
            {{- attrname }}="{{ attrvalue }}"
        {%- endif -%}
    {%- endfor -%}
{%- endblock attributes -%}